### DISCUSSION TITLE
deploying this on Fly.io isn't possible to be free anymore?

### DISCUSSION THREAD

**Discussion by ghost** [serverless-dns/serverless-dns#104] [Q&A]
![image](https://user-images.githubusercontent.com/72892766/209451922-f8ad9e39-8899-4e1c-b4e2-09dff0a3528c.png)

no matter what location I choose, it keeps asking for payment method. is there a way past this?

**Comment by ignoramous** [comment]
Nope, don't think so. Fly.io requires credit card on file to limit abuse apparently, see: https://archive.is/6qMah

Fly.io doesn't charge up to $5/mo, so their "free-tier" kind of generates invoices with 100% discount (and needs credit card on file): https://archive.is/zDitH
